docs/source/daml2ts/index.rst

```diff
@@ -31,15 +31,15 @@ Here's a complete example that generates TypeScript from a project produced from
    daml new my-proj skeleton # Create a new project based off the skeleton template
    cd my-proj # Enter the newly created project directory
    daml build  # Compile the project's DAML files into a DAR
-   daml codegen ts .daml/dist/my-proj-0.0.1.dar -o generated/ts --main-package-name=my-proj # Generate Typescript from the DAR
+   daml codegen ts .daml/dist/my-proj-0.0.1.dar -o daml2ts # Generate script bindings from the DAR
 
 - On execution of these commands:
 
-  - The directory ``my-proj/generated/ts`` contains the generated TypeScript source files;
+  - The directory ``my-proj/daml2ts`` contains generated TypeScript and Javascript artifacts;
   - The files are arranged into directories;
-  - One of those directories will be named as per the ``PACKAGE`` argument and will contain the TypeScript definitions corresponding to the DAML files in the project;
-  - For example, ``generated/ts/my-proj/Main.ts`` contains the TypeScript definitions for ``daml/Main.daml``;
-  - The remaining directories contain supporting TypeScript corresponding to modules of the DAML standard library;
+  - One of those directories will be named as per the ``PACKAGE`` argument and will contain the definitions corresponding to the DAML files in the project;
+  - For example, ``generated/ts/my-proj/Main.ts`` contains the definitions for ``daml/Main.daml``;
```

Comment:
Isn't there a `src` somewhere in the path to `Main.ts` as well?

Reply:
Yes. Fixed.